### PR TITLE
Added repairer type bodyshop for repairs

### PR DESCRIPTION
**Pull request by Yuri364**
| Q             | A
| ------------- | ---
| Bug fix?      | No
| New feature?  | Yes

### Added

- Example for `repairs` block with `Bodyshop` values for `repairs.history.items[].repairer.type`

Closes # (issue)

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I wrote unit tests for my code _(if tests is required for my changes)_
- [x] I have made changes in [CHANGELOG.md](https://github.com/avtocod/specs/blob/master/CHANGELOG.md) file

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog][keepachangelog] and this project adheres to [Semantic Versioning][semver].
 
+## v3.26.2
+
+### Added
+
+- Example for `repairs` block with `Bodyshop` values for `repairs.history.items[].repairer.type`
+
```
Comment (Reallife) [inline CHANGELOG.md:12]: Also was added new possible value for field with path `repairs.history.items[].repairer.type` and changed description of that field in `fields_list.json`. Add mentions about that in changelog, pls

Comment (Yuri364) [reply CHANGELOG.md:17]: Added

**Comment by tarampampam** [comment]
Bump patch (not minor)?

**Comment by Reallife** [comment]
> Bump patch (not minor)?
@tarampampam Добавили новое возможное значение. Думаю, что минор поднять надо всё-таки. 
@Yuri364 Исправь версию в чейнжлоге, пожалуйста.

**Pull request merged**